projects/plugins/GKNamesBookPlugin/GKNamesBookPlugin.net.csproj

```diff
@@ -2,6 +2,7 @@
 
     <PropertyGroup>
         <TargetFrameworks>net6.0</TargetFrameworks>
+        <RuntimeIdentifiers>linux-x64;linux-arm64</RuntimeIdentifiers>  

```

Comment:
It's not possible here. Besides Linux, the program is also available for Windows and MacOS.